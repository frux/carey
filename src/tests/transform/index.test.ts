import path from 'path';
import { assertMigration } from '../helpers';

describe('Actions', () => {
	describe('transform', () => {
		test('should transform file', () => assertMigration(
			{
				'foo.js': 'console.log(\'bar\');',
			},
			({ transform }) => ([
				{
					with: '/foo.js',
					do: [
						transform(path.resolve(__dirname, './fixtures/1.js')),
					],
				},
			]),
		));
	});
});
