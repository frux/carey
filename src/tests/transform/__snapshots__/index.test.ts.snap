// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actions transform should transform file 1`] = `
Object {
  "/foo.js": "console.log('baz');",
}
`;
